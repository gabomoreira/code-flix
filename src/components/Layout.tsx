import { Box, Container } from '@mui/material';
import React from 'react';

type Props = {
	children: React.ReactNode;
};

export const Layout = ({ children }: Props) => {
	return (
		<Box>
			<Container maxWidth="lg" sx={{ mt: 4, mb: 4, color: 'white' }}>
				{children}
			</Container>
		</Box>
	);
};
