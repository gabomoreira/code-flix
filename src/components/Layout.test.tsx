import { render } from '@testing-library/react';
import { Layout } from './Layout';

describe('Layout', () => {
	it('should render correctly', () => {
		const { asFragment } = render(
			<Layout>
				<div>Test</div>
			</Layout>
		);

		expect(asFragment()).toMatchSnapshot();
	});
});
