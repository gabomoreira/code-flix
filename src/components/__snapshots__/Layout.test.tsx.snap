// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render correctly 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-vooagt"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-87kmc0-MuiContainer-root"
    >
      <div>
        Test
      </div>
    </div>
  </div>
</DocumentFragment>
`;
